static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -59,11 +52,24 @@ const showJourneyAlert = (journeyEl: HTMLElement): void => {
             if (ev.isTrusted) {
                 ev.preventDefault();
                 Promise.all([
-                    shouldDisplayNewsletterWarning(journeyEl),
-                    shouldDisplayConsentWarning(journeyEl),
-                ]).then(([emptyNewsletters, emptyMarketing]) => {
-                    if (emptyNewsletters && emptyMarketing) {
-                        showModal('confirm-consents');
+                    missingNewsletters(journeyEl),
+                    missingConsents(journeyEl),
+                ]).then(([newslettersCheckboxInfo, marketingCheckboxInfo]) => {
+                    console.log(newslettersCheckboxInfo);
+                    if (
+                        newslettersCheckboxInfo.unchecked.length > 0 ||
+                        marketingCheckboxInfo.unchecked.length > 0
+                    ) {
+                        const allUncheckedBoxes = newslettersCheckboxInfo.unchecked.concat(
+                            marketingCheckboxInfo.unchecked
+                        );
+                        const names = allUncheckedBoxes.map(
```

Comment:
I thought this was only gonna list newsletters?

Reply:
Paula felt that marketing isn't given an equal chance to catch some users before they disappear into nowhere.

Reply:
Marketing want theirs in there as well